velox/functions/sparksql/tests/ArithmeticTest.cpp

```diff
@@ -401,5 +401,94 @@ TEST_F(LogNTest, log10) {
   EXPECT_EQ(log10(kInf), kInf);
 }
 
+class RandTest : public SparkFunctionBaseTest {
+ public:
+  RandTest() {
+    // This setting is needed as we expect int32 type is parsed for
```

Comment:
nit: Consider tightening up this comment. 

`Allow for parsing literal integers as INTEGER, not BIGINT.`